catroid/src/main/java/org/catrobat/catroid/utils/Utils.java

```diff
@@ -92,47 +88,15 @@ public static boolean isExternalStorageAvailable() {
 				&& !externalStorageState.equals(Environment.MEDIA_MOUNTED_READ_ONLY);
 	}
 
-	public static boolean isNetworkAvailable(Context context) {
-		ConnectivityManager connectivityManager = (ConnectivityManager) context
-				.getSystemService(Context.CONNECTIVITY_SERVICE);
-		return connectivityManager != null
-				&& connectivityManager.getActiveNetworkInfo() != null
-				&& connectivityManager.getActiveNetworkInfo().isConnected();
-	}
-
-	public static boolean checkForNetworkError(boolean success, WebconnectionException exception) {
-		return !success && exception != null && exception.getStatusCode() == WebconnectionException.ERROR_NETWORK;
-	}
-
-	public static boolean checkForSignInError(boolean success, WebconnectionException exception, Context context,
-			boolean userSignedIn) {
+	public static boolean checkForSignInError(boolean success, WebConnectionException exception, Context context,
```

Comment:
shouldn't you move this method to WebUtils as well?

Reply:
I'd like to move sign in and log out functionality into another class later

Reply:
I'd like to move sign in and log out functionality into another class later

Reply:
ok, I get that. one step at a time. :+1: